_posts/2023-09-05-Receive-mentorship-from-Amazon-engineers-and-accelerate-your-career-in-Tech.md

```diff
@@ -0,0 +1,60 @@
+
+<img src="/assets/media/blog-images/2023-09-05-college-contributor-initiative/You don't have to do it Alone.jpeg" alt="Not our first Rodeo"/>{: .img-fluid}
+## History of the program
+
+Earlier this year, from March 1st to May 15th, we launched the College Contributor Initiative (CCI), which was a pilot of OSCI. Well over 100 students participated in CCI, contributing an impressive 150 pull requests and generating 275 issues. Many participants have continued to work on the project, with multiple becoming maintainers.
+
+Please check out the **video** **testimonials** of some of the participants **[here](https://www.youtube.com/playlist?list=PLzgr9zSpws162v-8eNXgus5SFah73FjG6)**.
+
+>There were many aspects that differentiate this program from others. The weekly scheduled meetings with project maintainers and mentors were the most important method to speed up the contribution process. They allowed us to ask questions to clarify themes that were hindering the development process. They allowed us to get comfortable with a large code base in a short period of time
+
+OSCI is an expansion of CCI. While CCI included only a handful of institutions, OSCI will welcome multiple academic entities, coding bootcamps, professionals, enthusiasts, and anyone excited about open-source development.
+
+## What are the benefits of OSCI for participants?
+
+1. **Guidance from seasoned engineers**: You'll grow under the watchful eyes of expert engineers. It's the perfect space for learning and teaming up.
+2. **Real-world skill enhancement**: Dive into an impactful open-source project and sharpen your skills. Such experience can supercharge your career and portfolio. 
+3. **Constructive assessment of contributions**: All your contributions will be professionally assessed. You will have a chance to qualify to receive a recommendation letter from an AWS Principal Engineer or even an internal endorsement.
+
+## **Inviting everyone, everywhere**
+
+OSCI celebrates inclusivity. No boundaries. No borders. We welcome participants globally. Check out our [Code of Conduct](https://opensearch.org/codeofconduct.html) to see exactly how we are making the OpenSearch Project an inclusive and safe environment, or visit our [public Slack channel](https://opensearch.org/slack.html) to meet the community. 
+
+<img src="/assets/media/blog-images/2023-09-05-college-contributor-initiative/Rocket Leaving Earth.jpeg" alt="Kickstart your career"/>{: .img-fluid}
+## **In conclusion**
+
+OSCI will not only help you to grow your technical skills but will also allow you to connect with and learn from experienced Amazon engineers and community members. With guidance and resources at your fingertips, OSCI is poised to shape the builders of tomorrow.
+
+Eager to embark on this journey? **Apply now**: https://amazonmr.au1.qualtrics.com/jfe/form/SV_esNNNyGLItb5iPY
```

Comment:
you'll want to update so this is hyperlinked: 
`[https://amazonmr.au1.qualtrics.com/jfe/form/SV_esNNNyGLItb5iPY](https://amazonmr.au1.qualtrics.com/jfe/form/SV_esNNNyGLItb5iPY)`